4-star-nosed_mole/api_definitions/rest/ucs.yaml

```
@@ -101,13 +116,27 @@ paths:
           description: Multi-part upload successfully updated.
         "400":
           description: |
-            The provided status value was invalid or the status of this multi-part upload currently or permanently can't be changed.
+            Exceptions by ID:
+              - pendingInvalidChange: Cannot change status from pending to accepted or rejected
+              - uploadedInvalidChange: Cannot change status from uploaded to cancelled or pending
+              - invalidChangeFromCancelled: Cannot change status from cancelled for this upload attempt.
+                If you want to restart, initiate a new multi-part upload instead.
+              - invalidChangeFromFailed: Cannot change status from failed for this upload attempt.
+                If you want to restart, initiate a new multi-part upload instead.
+              - invalidChangeFromAccepted: Upload has already been accepted, a status change or
+                new upload attempt is no longer possible.
+              - invalidChangeFromRejected: Cannot change status from rejected for this upload attempt.
+                If you want to restart, initiate a new multi-part upload instead.
         "403":
           description: |
-            The user is not registered as a Data Submitter for the
-            file corresponding to this multi-part upload.
+            Exceptions by ID:
+              - userNotRegisteredUploadPatch: The user is not registered as a
+                Data Submitter for the file corresponding to this multi-part upload.
         "404":
-          description: The multi-part upload with the given ID does not exist.
+          description: |
+            Exceptions by ID:
+              - uploadNotExistingPatch: The multi-part upload with the given ID
```

Can be the same as above:

```suggestion
              - noSuchUpload: The multi-part upload with the given ID
```